Druid/DRIRGenerator.class.st

```diff
@@ -1806,6 +1808,8 @@ DRIRGenerator >> visitMessageNode: aRBMessageNode receiver: receiver [
 { #category : #visiting }
 DRIRGenerator >> visitMethodNode: aRBMethodNode [
 
+	(aRBMethodNode hasPragmaNamed: #doNotGenerate) ifTrue: [ 1 halt ].
```

Comment:
Interesting, have a case?
Why not the same pragma that in the other PR for the halt?

Reply:
Hahahah I used it to know quickly where the meta-compilation scapes from the "real" interpreter code.

Good to find missing special cases. We can remove it (or not... should never happen I think 🤔 )